components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
I'm afraid with a large SVG DOM may be visibly slow - but that needs to be tested (in general from my experience DOM manipulation is MUCH slower than text manipulation, while being safer/less prone to mistakes, of course - the slow part is not to clone, but to do actual modifications) - especially as for say print preview we need effectively N copies of SVG, with different viewports (then again, can do an in-memory copy, do DOM manipulations, and only change vieports as text)